scipy/cluster/hierarchy.py

```diff
@@ -268,6 +268,63 @@ def single(y):
     linkage: for advanced creation of hierarchical clusterings.
     scipy.spatial.distance.pdist : pairwise distance metrics
 
+    Examples
+    --------
+    >>> from scipy.cluster.hierarchy import single, fcluster
+    >>> from scipy.spatial.distance import pdist
+
+    First we need a toy dataset to play with:
+
+    x x    x x
```

Comment:
I don't know if these `x`s are meant to provide ASCII art but if so they need to be indented. Currently they don't render correctly as codeblocks

See https://6690-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.cluster.hierarchy.single.html#scipy.cluster.hierarchy.single

Reply:
... and hte previous line needs to end with double colon `::`,
http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html